php-http/httplug-bundle/1.6/config/packages/httplug.yaml

```diff
@@ -0,0 +1,14 @@
+httplug:
+    plugins:
+        redirect:
+            preserve_header: true
+
+    discovery:
+        client: 'auto'
+
+    clients:
+        app:
+            http_methods_client: true
+            plugins:
+                - 'httplug.plugin.content_length'
```

Comment:
@Nyholm the contrib recipe was enabling the logger plugin. Is is expected to remove it here ?

Reply:
Yes, I removed logger plugin since it depends on Monolog.